### ISSUE TITLE
Feature request, focused window after it's drawn.

### ISSUE THREAD

**Issue by freed00m** [vladimiry/email-securely-app#43]
Hi,

I like your app, would it be possible for protonmail tab have the inner window focused?

I can't use the protonmail's J, K keys for navigation right after the window pops up. I have to use mouse to click anywhere inside the protonmail window view to have J, K keyboard navigation.

I use Arch linux, so I cannot comment about other platforms.

**Comment by vladimiry** [comment]
Does [this](https://www.dropbox.com/s/xxay30m7ruu9ymj/email-securely-app-1.4.1-linux.pacman?dl=1) version fulfill the need?

**Comment by freed00m** [comment]
@vladimiry hi, I've tested your binary and must say it works only the first time, repeatingly toggling the windows breaks the behavior.

**Comment by vladimiry** [comment]
Thanks for testing. What about [this](https://www.dropbox.com/s/8t8yn9g7wltu6z6/email-securely-app-1.4.1-linux.pacman?dl=0) version? Please share your complete testing actions sequence if the provided version still doesn't work as expected.

**Comment by freed00m** [comment]
@vladimiry YES, this works perfectly. Will you include it in the next release? :) :+1:

If you do, you can close this issue. 

```
~ (M) $ t
/tmp $ md5sum email-securely-app-1.4.1-linux.pacman 
15a3798cfee79b525d4bc669983f9567  email-securely-app-1.4.1-linux.pacman  
/tmp $ sudo pacman -U email-securely-app-1.4.1-linux.pacman 
[sudo] password for frdm: 
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (12) gtk-sharp-2-2.12.45-1  libappindicator-gtk2-12.10.0-12  libappindicator-gtk3-12.10.0-12  libappindicator-sharp-12.10.0-12  libdbusmenu-glib-16.04.0-2
              libdbusmenu-gtk2-16.04.0-2  libdbusmenu-gtk3-16.04.0-2  libgdiplus-5.6-1  libindicator-gtk2-12.10.1-7  libindicator-gtk3-12.10.1-7  mono-5.14.0.177-1
              email-securely-app-1.4.1-1

Total Installed Size:  395.81 MiB

:: Proceed with installation? [Y/n] 
(12/12) checking keys in keyring                                                                        [##############################################################] 100%
(11/12) checking package integrity                                                                      [##############################################################] 100%
(11/12) loading package files                                                                           [##############################################################] 100%
(12/12) checking for file conflicts                                                                     [##############################################################] 100%
(12/12) checking available disk space                                                                   [##############################################################] 100%
:: Processing package changes...
( 1/12) installing libdbusmenu-glib                                                                     [##############################################################] 100%
( 2/12) installing libdbusmenu-gtk2                                                                     [##############################################################] 100%
( 3/12) installing libindicator-gtk2                                                                    [##############################################################] 100%
( 4/12) installing libappindicator-gtk2                                                                 [##############################################################] 100%
( 5/12) installing libdbusmenu-gtk3                                                                     [##############################################################] 100%
( 6/12) installing libindicator-gtk3                                                                    [##############################################################] 100%
( 7/12) installing libappindicator-gtk3                                                                 [##############################################################] 100%
( 8/12) installing libgdiplus                                                                           [##############################################################] 100%
( 9/12) installing mono                                                                                 [##############################################################] 100%
Mono Certificate Store Sync - version 5.14.0.0
Populate Mono certificate store from a concatenated list of certificates.
Copyright 2002, 2003 Motus Technologies. Copyright 2004-2008 Novell. BSD licensed.

Importing into legacy system store:
I already trust 135, your new list has 134
Import process completed.

Importing into BTLS system store:
I already trust 135, your new list has 134
Import process completed.
(10/12) installing gtk-sharp-2                                                                          [##############################################################] 100%
(11/12) installing libappindicator-sharp                                                                [##############################################################] 100%
(12/12) installing email-securely-app                                                                   [##############################################################] 100%
:: Running post-transaction hooks...
(1/4) Updating icon theme caches...
(2/4) Registering binary formats...
(3/4) Arming ConditionNeedsUpdate...
(4/4) Updating the desktop file MIME type cache...
/tmp $ email-securely-app 
```

**Comment by vladimiry** [comment]
Released with [v1.4.1](https://github.com/vladimiry/email-securely-app/releases/tag/v1.4.1).
